### PR TITLE
gsdx-ogl: invert behavior of unsafe user hack

### PR DESCRIPTION
**Pull request by gregory38**
It would be on by default. Unsafe & fast path.

The hack is a security if someone encounters any issue

### CODE REVIEW

File: plugins/GSdx/resource.h [hunk 1]
```diff
@@ -89,7 +89,7 @@
 #define IDC_MIPMAP                      2084
 #define IDC_PRELOAD_GS                  2085
 #define IDC_TVSHADER                    2086
-#define IDC_UNSAFE_FBMASK               2087
+#define IDC_SAFE_FBMASK               2087
```
Comment (ssakash) [inline plugins/GSdx/resource.h]: ``2087`` needs alignment over here.

**Comment by ssakash** [comment]
one more thing: the hack is called "Fast Accurate blending hack" so users would normally expect to get a speedup when checking the box. It'd be better to change the checkbox text (or) set the default value to 1.

File: plugins/GSdx/GSLinuxDialog.cpp [hunk 2]
```diff
@@ -351,7 +351,7 @@ void populate_hack_table(GtkWidget* hack_table)
 	GtkWidget* hack_tco_entry      = CreateTextBox("UserHacks_TCOffset");
 	GtkWidget* align_sprite_check  = CreateCheckBox("Align sprite hack", "UserHacks_align_sprite_X");
 	GtkWidget* preload_gs_check    = CreateCheckBox("Preload Frame", "preload_frame_with_gs_data");
-	GtkWidget* hack_unsafe_fbmask  = CreateCheckBox("Fast Accurate Blending", "UserHacks_unsafe_fbmask");
+	GtkWidget* hack_safe_fbmask    = CreateCheckBox("Fast Accurate Blending", "UserHacks_safe_fbmask");
```
Comment (turtleli) [inline plugins/GSdx/GSLinuxDialog.cpp]: Fast -> Safe

File: plugins/GSdx/GSSetting.cpp [hunk 3]
```diff
@@ -102,9 +102,8 @@ const char* dialog_message(int ID, bool* updateText) {
 				"High:\nExtend it to destination alpha blending and color wrapping. (help shadow and fog effect). A good CPU is required.\n\n"
 				"Full:\nExcept few cases, the blending unit will be fully emulated by the shader. It is ultra slow! It is intended for debug.\n\n"
 				"Ultra:\nThe blending unit will be completely emulated by the shader. It is ultra slow! It is intended for debug.";
-		case IDC_UNSAFE_FBMASK:
-			return "Relies on undefined hardware behavior to accelerate accurate blending operation.\n\n"
-				"It provides a major boost on the Xenosaga Series.\n";
+		case IDC_SAFE_FBMASK:
+			return "By default, accurate blending relies on undefined hardware behavior to be fast.\nThis option enable a slower but safer behavior if anyone encouters an issue.\n";
```
Comment (turtleli) [inline plugins/GSdx/GSSetting.cpp]: enable -> enables
encouter -> encounter

File: plugins/GSdx/GSLinuxDialog.cpp (hunk 2 above)
Comment (ssakash) [inline plugins/GSdx/GSLinuxDialog.cpp]: would be better changing it to ``Safe accurate blending``. the checkbox text  for windows doesn't use a uppercase letter for ``Accurate`` and ``Blending``.

Comment (gregory38) [inline plugins/GSdx/GSLinuxDialog.cpp]: I like uppercase ;)

**Pull request merged**